src/index.js

```diff
@@ -1,13 +1,13 @@
-import React from 'react'
-import { render } from 'react-dom'
-import { Provider } from 'react-redux'
-import { ConnectedRouter } from 'react-router-redux'
-import store, { history } from './store'
-import App from './containers/app'
+import React from 'react';
+import { render } from 'react-dom';
+import { Provider } from 'react-redux';
+import { ConnectedRouter } from 'react-router-redux';
+import store, { history } from './store';
+import App from './containers/app';
 
-import './index.css'
+import './index.css';
 
-const target = document.querySelector('#root')
+const target = document.querySelector('#root');
```

Comment:
why are you using `document.querySelector`?

Reply:
We peer-programmed this part on Friday and we found it online on how to setup a react app.

Reply:
Check with Kimi, but I'll approve for now